pkg/apis/policy/v1alpha1/clusterimagepolicy_validation.go

```diff
@@ -154,19 +160,15 @@ func (key *KeyRef) Validate(ctx context.Context) *apis.FieldError {
 
 func (keyless *KeylessRef) Validate(ctx context.Context) *apis.FieldError {
 	var errs *apis.FieldError
-	if keyless.URL == nil && keyless.Identities == nil && keyless.CACert == nil {
-		errs = errs.Also(apis.ErrMissingOneOf("url", "identities", "ca-cert"))
+	if keyless.URL == nil && keyless.CACert == nil {
+		errs = errs.Also(apis.ErrMissingOneOf("url", "ca-cert"))
```

Comment:
Why is the validation for `identities` being removed?

Reply:
It's not a required field. Just like with cosign the CLI if you do not specify `--certificate-email` or `--certificate-oidc-issuer` the signature / attestation is verified, but it's not checked who signed it.
Identities are still validated if they are specified.